Updating `otherlibs/win32unix/sockopt.c` somehow broke the CI:
```c
sockopt.c:50:17: error: ‘SO_REUSEPORT’ undeclared here (not in a function); did you mean ‘SO_REUSEADDR’?
   { SOL_SOCKET, SO_REUSEPORT },
                 ^~~~~~~~~~~~
                 SO_REUSEADDR
```
Maybe Cygwin doesn't support this option. Reverting.

--- a/Changes
+++ b/Changes
@@ -204,9 +204,6 @@ Working version
 - #9575: Add Unix.is_inet6_addr
   (Nicolás Ojeda Bär, review by Xavier Leroy)
 
-- #9869: Add Unix.SO_REUSEPORT
-  (Yishuai Li)
-
 - #9593: Use new flag for non-elevated symbolic links and test for Developer
   Mode on Windows
   (Manuel Hornung, review by David Allsopp and Nicolás Ojeda Bär)
@@ -221,6 +218,9 @@ Working version
 
 - #9802: Ensure signals are handled before Unix.kill returns
   (Stephen Dolan, review by Jacques-Henri Jourdan)
+
+- #9869: Add Unix.SO_REUSEPORT
+  (Yishuai Li, review by Xavier Leroy)
 
 ### Tools:
 

--- a/Changes
+++ b/Changes
@@ -204,6 +204,9 @@ Working version
 - #9575: Add Unix.is_inet6_addr
   (Nicolás Ojeda Bär, review by Xavier Leroy)
 
+- #9869: Add Unix.SO_REUSEPORT
+  (Yishuai Li)
+
 - #9593: Use new flag for non-elevated symbolic links and test for Developer
   Mode on Windows
   (Manuel Hornung, review by David Allsopp and Nicolás Ojeda Bär)

--- a/otherlibs/unix/sockopt.c
+++ b/otherlibs/unix/sockopt.c
@@ -38,6 +38,9 @@
 #ifndef SO_REUSEADDR
 #define SO_REUSEADDR (-1)
 #endif
+#ifndef SO_REUSEPORT
+#define SO_REUSEPORT (-1)
+#endif
 #ifndef SO_KEEPALIVE
 #define SO_KEEPALIVE (-1)
 #endif
@@ -109,6 +112,7 @@ static struct socket_option sockopt_bool[] = {
   { SOL_SOCKET, SO_DEBUG },
   { SOL_SOCKET, SO_BROADCAST },
   { SOL_SOCKET, SO_REUSEADDR },
+  { SOL_SOCKET, SO_REUSEPORT },
   { SOL_SOCKET, SO_KEEPALIVE },
   { SOL_SOCKET, SO_DONTROUTE },
   { SOL_SOCKET, SO_OOBINLINE },

--- a/otherlibs/unix/unix.ml
+++ b/otherlibs/unix/unix.ml
@@ -597,6 +597,7 @@ type socket_bool_option =
     SO_DEBUG
   | SO_BROADCAST
   | SO_REUSEADDR
+  | SO_REUSEPORT
   | SO_KEEPALIVE
   | SO_DONTROUTE
   | SO_OOBINLINE

--- a/otherlibs/unix/unix.mli
+++ b/otherlibs/unix/unix.mli
@@ -1429,6 +1429,7 @@ type socket_bool_option =
     SO_DEBUG       (** Record debugging information *)
   | SO_BROADCAST   (** Permit sending of broadcast messages *)
   | SO_REUSEADDR   (** Allow reuse of local addresses for bind *)
+  | SO_REUSEPORT   (** Allow reuse of address and port bindings *)
   | SO_KEEPALIVE   (** Keep connection active *)
   | SO_DONTROUTE   (** Bypass the standard routing algorithms *)
   | SO_OOBINLINE   (** Leave out-of-band data in line *)

--- a/otherlibs/unix/unixLabels.mli
+++ b/otherlibs/unix/unixLabels.mli
@@ -1175,6 +1175,7 @@ type socket_bool_option =
     SO_DEBUG       (** Record debugging information *)
   | SO_BROADCAST   (** Permit sending of broadcast messages *)
   | SO_REUSEADDR   (** Allow reuse of local addresses for bind *)
+  | SO_REUSEPORT   (** Allow reuse of address and port bindings *)
   | SO_KEEPALIVE   (** Keep connection active *)
   | SO_DONTROUTE   (** Bypass the standard routing algorithms *)
   | SO_OOBINLINE   (** Leave out-of-band data in line *)

--- a/otherlibs/win32unix/sockopt.c
+++ b/otherlibs/win32unix/sockopt.c
@@ -21,6 +21,9 @@
 #include "unixsupport.h"
 #include "socketaddr.h"
 
+#ifndef SO_REUSEPORT
+#define SO_REUSEPORT (-1)
+#endif
 #ifndef IPPROTO_IPV6
 #define IPPROTO_IPV6 (-1)
 #endif
@@ -47,6 +50,7 @@ static struct socket_option sockopt_bool[] = {
   { SOL_SOCKET, SO_DEBUG },
   { SOL_SOCKET, SO_BROADCAST },
   { SOL_SOCKET, SO_REUSEADDR },
+  { SOL_SOCKET, SO_REUSEPORT },
   { SOL_SOCKET, SO_KEEPALIVE },
   { SOL_SOCKET, SO_DONTROUTE },
   { SOL_SOCKET, SO_OOBINLINE },

--- a/otherlibs/win32unix/unix.ml
+++ b/otherlibs/win32unix/unix.ml
@@ -729,6 +729,7 @@ type socket_bool_option =
     SO_DEBUG
   | SO_BROADCAST
   | SO_REUSEADDR
+  | SO_REUSEPORT
   | SO_KEEPALIVE
   | SO_DONTROUTE
   | SO_OOBINLINE

